tests/src/test/java/alluxio/server/configuration/ConfigCheckerIntegrationTest.java

```diff
@@ -74,6 +74,26 @@ public void multiMasters() throws Exception {
     mCluster.notifySuccess();
   }
 
+  @Test
+  public void multiMastersEmbeddedHA() throws Exception {
+    PropertyKey key = PropertyKey.MASTER_JOURNAL_FLUSH_TIMEOUT_MS;
+    Map<Integer, Map<PropertyKey, String>> masterProperties
+        = generatePropertyWithDifferentValues(TEST_NUM_MASTERS, key);
+    mCluster = MultiProcessCluster.newBuilder(PortCoordination.CONFIG_CHECKER_MULTI_MASTERS)
+        .setClusterName("ConfigCheckerMultiMastersEmbeddedHATest")
+        .setNumMasters(TEST_NUM_MASTERS)
+        .setNumWorkers(0)
+        .setDeployMode(DeployMode.EMBEDDED_HA)
+        .setMasterProperties(masterProperties)
+        .build();
+    mCluster.start();
+    ConfigCheckReport report = getReport();
+    assertEquals(ConfigStatus.FAILED, report.getConfigStatus());
```

Comment:
why is this status `FAILED` while the status on line 71 is `WARN`?

Reply:
The `alluxio.master.journal.folder` is different between master processes and this property has the consistency check level as `ENFORCE`

Reply:
I see - can you add a comment for this? It might also be confusing to future maintainers